Source/Core/Core/HW/EXI/EXI_Device.h

```diff
@@ -37,6 +37,7 @@ enum class EXIDeviceType : int
   EthernetXLink,
   // Only used on Apple devices.
   EthernetTapServer,
+  BuiltIn,
```

Comment:
This should be called `EthernetBuiltIn`, as otherwise it's unclear what it's referring to (the first thing I would think of when asked about a built-in EXI device is the mask ROM and other gamecube stuff that's internal and always connected).